### PR TITLE
add a make commands that start phase1

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

**Comment by errordeveloper** [comment]
LGTM

**Comment by colemickens** [comment]
Should it use something like `./$$(jq -r '.phase1.instance_prefix')/` as the staging directory rather than `./tmp`? If I modify `phase1.instance_prefix` now and re-deploy, it will pick up the old terraform file and tear down my old cluster.

I was thinking about evaluating `$$(jq -r '.phase1.instance_prefix' .config.json)` in the top Makefile and passing it as an extra argument so that `do [deploy|destroy]` becomes `do [deploy|destroy] {instance_prefix}` and then uses that arg as the staging dir?

**Comment by errordeveloper** [comment]
@colemickens that's a good question indeed. Terraform does a good job at figuring out when resources need to be updated vs re-created entierely, but there is probably something we should do outside of Terraform, especially to cater for non-Terraform implementations of phase 1. I think instance prefix, or cluster name would be good candidates for that. I also think that generated code in the sense we have it right now should be eligible to a life of it's own, i.e. users might need to modify and redistribute the generated code.

File: Makefile [hunk 1]
```diff
@@ -45,6 +45,14 @@ menuconfig: .tmp/mconf
 echo-config: .config.json
 	cat $<
 
+deploy: .config.json
```
Comment (errordeveloper) [inline Makefile]: I suppose you can do:
```
deploy destroy: .config.json
 	$(MAKE) do WHAT=$@
```
Isn't it?

Comment (mikedanese) [inline Makefile]: Done.

**Comment by mikedanese** [comment]
@colemickens compromise: name the tf files differently based on instance prefix which is basically cluster name. I pushed a change to all.jsonnet that does that.

**Comment by colemickens** [comment]
That doesn't include the crypto assets then. I think it would be good for there to be an `output` folder from the deployment that contains a `kubeconfig`, the terraform plan, the terraform state and the crypto assets. Maybe I'll just file a separate issue.

**Comment by mikedanese** [comment]
Ouch, the crypto stuff is definitely not threadsafe right now... Ok I'm going to work on https://github.com/kubernetes/kubernetes-anywhere/issues/145 next.

**Comment by mikedanese** [comment]
I'm going to leave that work for a follow up.

**Pull request merged**